frontend/src/routes/candidate/password-reset/PasswordResetPage.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import {page} from '$app/stores';
+  import {goto} from '$app/navigation';
+  import {resetPassword} from '$lib/api/candidate';
+  import {FrontPage} from '$lib/templates/frontPage';
+  import {candidateAppRoute} from '$candidate/placeholder.json';
+  import {HeadingGroup, PreHeading} from '$lib/components/headingGroup';
+  import {PasswordValidator} from '$candidate/components/passwordValidator';
+  import {validatePassword} from '$lib/utils/passwordValidation';
+  import {Button} from '$lib/components/button';
+  import Footer from '$lib/components/footer/Footer.svelte';
+
+  export let code: string;
+
+  let password = '';
+  let passwordConfirmation = '';
+  let validPassword = false;
+  let errorMessage = '';
+
+  $: disableSetButton = validPassword && passwordConfirmation.length > 0;
+
```

Comment:
Should this be named differently? Now it is `false` when the button should be disabled.